wiki/president_place_of_birth_tutorial.ipynb

```diff
@@ -0,0 +1,1638 @@
+      " 'Andrew_Jackson',\n",
+      " 'George_Washington',\n",
+      " 'Al_Gore',\n",
+      " 'James_K._Polk',\n",
+      " 'James_A._Garfield',\n",
+      " 'Bill_Clinton',\n",
+      " 'George_M._Dallas',\n",
+      " 'Aaron_Burr',\n",
+      " 'James_Buchanan',\n",
+      " 'John_Nance_Garner',\n",
+      " 'Barack_Obama',\n",
+      " 'Herbert_Hoover',\n",
+      " 'John_C._Calhoun',\n",
+      " 'Benjamin_Harrison',\n",
+      " 'George_W._Bush',\n",
+      " 'John_F._Kennedy',\n",
+      " 'Franklin_Pierce',\n",
+      " 'John_C._Breckinridge',\n",
+      " 'George_Clinton',\n",
+      " 'John_Tyler',\n",
+      " 'Garret_Hobart',\n",
+      " 'Levi_P._Morton',\n",
+      " 'John_Quincy_Adams',\n",
+      " 'Dick_Cheney',\n",
+      " 'Alben_W._Barkley',\n",
+      " 'Adlai_Stevenson',\n",
+      " 'Donald_Trump']\n"
+     ]
+    }
+   ],
+   "source": [
+    "docs = session.query(Document).order_by(Document.name).all()\n",
+    "ld   = len(docs)\n",
+    "\n",
+    "train_docs = set()\n",
+    "dev_docs   = set()\n",
+    "test_docs  = set()\n",
+    "splits = (0.7, 0.85)\n",
+    "data = [(doc.name, doc) for doc in docs]\n",
+    "data.sort(key=lambda x: x[0])\n",
+    "for i, (doc_name, doc) in enumerate(data):\n",
+    "    if i < splits[0] * ld:\n",
+    "        train_docs.add(doc)\n",
+    "    elif i < splits[1] * ld:\n",
+    "        dev_docs.add(doc)\n",
+    "    else:\n",
+    "        test_docs.add(doc)\n",
+    "from pprint import pprint\n",
+    "pprint([x.name for x in train_docs])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Phase 2: Mention Extraction, Candidate Extraction Multimodal Featurization\n",
+    "\n",
+    "Given the unified data model from Phase 1, `Fonduer` extracts relation\n",
+    "candidates based on user-provided **matchers** and **throttlers**. Then,\n",
+    "`Fonduer` leverages the multimodality information captured in the unified data\n",
+    "model to provide multimodal features for each candidate.\n",
+    "\n",
+    "## 2.1 Mention Extraction\n",
+    "\n",
+    "The first step is to extract **mentions** from our corpus. A `mention` is the\n",
+    "type of object which makes up a `candidate`. For example, if we wanted to\n",
+    "extract pairs of (vice) president names and their corresponding place of birth, the name would be one `mention` while\n",
+    "the place of birth would be another. These `mention`s are then combined to\n",
+    "create `candidates`, where our task is to predict which `candidates` are true\n",
+    "in the associated document.\n",
+    "\n",
+    "We first start by defining and naming our two `mention`s:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from fonduer.candidates.models import mention_subclass\n",
+    "\n",
+    "Presidentname = mention_subclass(\"Presidentname\")\n",
+    "Placeofbirth = mention_subclass(\"Placeofbirth\")\n"
```

Comment:
Our [docstring](https://fonduer.readthedocs.io/en/latest/user/candidates.html#fonduer.candidates.models.mention_subclass) suggests that these be camel case:
```
The name of the class, should be “camel case” e.g. NewMention
```
So `PresidentName` and `PlaceOfBirth`. This doesn't actually affect functionality, but if you wanted to switch it to be more in line with the docstring, I also think the CamelCase makes it a little easier to read.